x-pack/plugins/enterprise_search/server/lib/ml/fetch_ml_models.ts

```diff
@@ -154,7 +154,7 @@ const enrichModelWithDownloadStatus = async (
 
   if (modelConfigWithDefinitionStatus && modelConfigWithDefinitionStatus.count > 0) {
     model.deploymentState = modelConfigWithDefinitionStatus.trained_model_configs[0].fully_defined
-      ? MlModelDeploymentState.Downloaded
+      ? MlModelDeploymentState.NotDeployed
```

Comment:
Once a model is downloaded, its status should change to "Not started". This is the condition for displaying the Start button.

This fetch logic is only used by the model selection list, so there are no side effects.